### DISCUSSION TITLE
Missing VFS libarchive addon for Kodi

### DISCUSSION THREAD

**Discussion by Thundervarela** [MichaIng/DietPi#5966] [Q&A] [answered]
This is the offical VFS libarchive that is missing on kodi 19.4 on lastest versions that comes packed on dietpi https://debian.pkgs.org/11/debian-main-arm64/kodi-vfs-libarchive_2.0.1+ds1-1_arm64.deb.html please fix this problem because without it isn't posible to install the subtitles from legendasDivx.com

Thank you for your atencion

**Comment by Joulinar** [comment, marked as answer]
Can you share some more information on your system. For some systems Kodi is installed from office source/apt repository.

**Reply by Thundervarela** [reply]
dietpi 8.11.2 and kodi 19.4 and when i go to install the subtitles from legendasdivx.com it needs this lib and the lib it's not available

**Reply by Thundervarela** [reply]
well it's arm64 on raspberry pi 4 64 bits 8gb ram

**Reply by Joulinar** [reply]
simply install the package if it is missing using `apt install kodi-vfs-libarchive`

**Reply by Thundervarela** [reply]
didn't install the output : 

```
sudo apt install kodi-vfs-libarchive
Reading package lists... Done
Building dependency tree... Done
Reading status information... Done
The following additional packages will be installed:
   libarchive13
Suggested packages:
   lrzip
The following NEW packages will be installed:
   kodi-vfs-libarchive libarchive13
0 packages updated, 2 new packages installed, 0 to remove and 0 not updated.
You need to get 414 kB of files.
After this operation, an additional 1430 kB of disk space will be used.
Do you wish to continue? [Y/n] Y
Get:1 https://deb.debian.org/debian bullseye/main arm64 libarchive13 arm64 3.4.3-2+deb11u1 [320 kB]
Obtain:2 https://archive.raspberrypi.org/debian bullseye/main arm64 kodi-vfs-libarchive arm64 19.2.0-3~bullseye [94.6 kB]
Obtained 414 kB in 1s (503 kB/s)
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package libarchive13:arm64.
(Reading database... 27036 files and directories currently installed.)
Preparing to unpack .../libarchive13_3.4.3-2+deb11u1_arm64.deb ...
Unpacking libarchive13:arm64 (3.4.3-2+deb11u1) ...
Selecting previously unselected package kodi-vfs-libarchive.
Preparing to unpack .../kodi-vfs-libarchive_19.2.0-3~bullseye_arm64.deb ...
Unpacking kodi-vfs-libarchive (19.2.0-3~bullseye) ...
Installing libarchive13:arm64 (3.4.3-2+deb11u1) ...
Installing kodi-vfs-libarchive (19.2.0-3~bullseye) ...
Processing triggers for libc-bin (2.31-13+rpt2+rpi1+deb11u5) ...
```

**Reply by Joulinar** [reply]
there is no error message on your output

**Reply by Thundervarela** [reply]
Fixed with reboot thanks

**Reply by Thundervarela** [reply]
With reboot after the command sudo apt install kodi-vfs-libarchive it got fix that lib

**Comment by MichaIng** [comment]
How does it not work? Did you restart Kodi?

Not sure if this is the issue, but while Kodi from RPi repo is v19.4, the kodi-vfs-libarchive package is v19.2. Both were however built on the exact same day, so it at least looks like it's expected. Ah and also on Debian the version strings do not match. Seems like the Kodi major version alignment has been done just recently: On Bullseye backports it's v19.4 vs v19.0.

EDIT: Ah solved, okay them 🙂.

**Reply by Thundervarela** [reply]
Brand new fresh install and those are the issues

**Reply by MichaIng** [reply]
Jep, so no issue at all, just a missing package for an additional feature.

**Reply by Thundervarela** [reply]
issue is this one : https://github.com/MichaIng/DietPi/discussions/5968

**Reply by Joulinar** [reply]
don't mix the topics and continue on the other one.

**Reply by Thundervarela** [reply]
sorry
